examples/resources/network_policy/resource_organization_custom-ingress_network_policy.tf

```diff
@@ -0,0 +1,128 @@
+/*
+Organization scoped Tanzu Mission Control network policy with custom-ingress input recipe.
+This policy is applied to a organization with the custom-ingress configuration option.
+The defined scope and input blocks can be updated to change the policy's scope and recipe, respectively.
+*/
+resource "tanzu-mission-control_network_policy" "organization_scoped_custom-ingress_source-selector_network_policy" {
+  name = "tf-network-test"
+
+  scope {
+    organization {
+      organization = "dummy-id"
+    }
+  }
+
+  spec {
+    input {
+      custom_ingress {
+        rules {
+          ports {
+            port     = "8443"
+            protocol = "TCP"
+          }
+          rule_spec {
+            custom_selector {
+              namespace_selector {
+                key   = "key1a"
```

Comment:
Since customer are using these examples and most of the time, they just copy paste it. 
so lets start providing example from real world scenario, ie copy paste of this terraform block should work in real scenario.

Reply:
These examples are exactly the same resource blocks that I used in the live working demo, hence these are tested and work perfectly.

Reply:
Key's value should be component right ? that is what was shown in demo

Reply:
The examples should have real values which will be used in production

Reply:
> Key's value should be component right ? that is what was shown in demo

That namespace selector block is different and common to all policies and present in all scripts after the spec for this recipe ends. Check lines: 55-69

Reply:
> The examples should have real values which will be used in production

These are generic values and can be anything, so added the value as generic as possible. This has validation on schema as a validRegex. Also, resource templates have placeholders that give better description of what needs to be used for each spec value.

Reply:
Instead of key1a and value1a, can you add the right namespace selector

Reply:
I believe these should be fine since it is just key and value not governed by any rules but the length and regex that has been validated on the schema level. Still, I have asked the policy team to give some example values. 
cc: @sbasu31